employees/views.py

```diff
@@ -157,22 +159,9 @@ def redirect_to_current_month(self) -> HttpResponseRedirectBase:
         return redirect(self._get_current_month_url(pk))
 
 
-@method_decorator(login_required, name="dispatch")
-@method_decorator(
-    check_permissions(
-        allowed_user_types=[
-            CustomUser.UserType.EMPLOYEE.name,
-            CustomUser.UserType.MANAGER.name,
-            CustomUser.UserType.ADMIN.name,
-        ]
-    ),
-    name="dispatch",
-)
```

Comment:
What about these permission decorators? Without it somebody else can access your reports